### ISSUE TITLE
"Callback must be a function" error

### ISSUE THREAD

**Issue by ibruynin** [meridius/confluence-to-markdown#7]
Hi @meridius 

I'm trying to run this in an azure devops build pipeline (after I tried it manually on another host)
This (cfr. below) is what I see in the build output and also when trying to execute it manually afterwards.

Any idea what's going wrong and how to resolve it?
Thanks in advance.

```
C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master>npm run start ..\..\Confluence-export ..\..\Markdown

> confluence-to-markdown@1.1.2 start C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master
> coffee ./src/index.coffee "..\..\Confluence-export" "..\..\Markdown"

Using source: C:\agent\_work\2\s\Confluence-export
Using destination: C:\agent\_work\2\s\Markdown
TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at makeCallback (fs.js:137:11)
    at Object.unlink (fs.js:917:14)
    at App.writeMarkdownFile (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\App.coffee:94:10)
    at App.writeGlobalIndexFile (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\App.coffee:105:6)
    at App.convert (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\App.coffee:55:6)
    at Bootstrap.run (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\Bootstrap.coffee:34:9)
    at Object.<anonymous> (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\index.coffee:7:11)
    at Object.<anonymous> (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\src\index.coffee:1:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.exports.run (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\node_modules\coffee-script\lib\coffee-script\coffee-script.js:173:23)
    at compileScript (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\node_modules\coffee-script\lib\coffee-script\command.js:224:29)
    at compilePath (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\node_modules\coffee-script\lib\coffee-script\command.js:174:14)
    at Object.exports.run (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\node_modules\coffee-script\lib\coffee-script\command.js:98:20)
    at Object.<anonymous> (C:\agent\_work\2\s\confluence-to-markdown\confluence-to-markdown-master\node_modules\coffee-script\bin\coffee:15:45)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:752:3)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! confluence-to-markdown@1.1.2 start: `coffee ./src/index.coffee "..\..\Confluence-export" "..\..\Markdown"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the confluence-to-markdown@1.1.2 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\***\AppData\Roaming\npm-cache\_logs\2018-10-28T22_35_12_048Z-debug.log
```

**Comment by ibruynin** [comment]
Update: seems node v10 related
Solution: in App.coffee:94 change

    @_fs.unlink tempInputFile

to

    @_fs.unlinkSync tempInputFile

tested on node v6 and that still works

**Comment by meridius** [comment]
closed by #6
